sass/project.sass

```diff
@@ -73,6 +150,36 @@
 //     &--left
 //       color: red
 
+
+.svg-streak
+  +span-columns(2 of 12)
+  +shift(5)
+  @media screen and (min-width: 768px)
+    +span-columns(2 of 12)
+    +shift(5)
+  position: absolute
+  left: 0
+  //transform: translateY(-125rem)
+  top: -15rem
+  z-index: -1
+
+
 /*! helper classes for layout */
 .right
   float: right
+
+.container--padding
+  margin-bottom: 5rem
+  @media screen and (min-width: 768px)
+    margin-bottom: 10rem
+
+.layout__background-color
+  height: 40rem
+  width: 100%
+  position: absolute
+  top: 0
+  left: 0
+  z-index: -1
```

Comment:
This class is fun cause its named for `background-color` but there is no mention of color at all